server/base/src/main/java/org/apache/accumulo/server/security/handler/KerberosAuthenticator.java

```diff
@@ -190,9 +188,4 @@ public synchronized boolean userExists(String user) {
     return SUPPORTED_TOKENS;
   }
 
-  @Override
-  public boolean validTokenClass(String tokenClass) {
-    return SUPPORTED_TOKEN_NAMES.contains(tokenClass);
```

Comment:
I think this was intended as part of the design to eventually support multi-tiered authentication, just like the `passwd` entry in Linux's `/etc/nsswitch.conf` can include an ordered list of authentication storage mechanisms or (at a higher level) PAM can be configured to use multiple authentication modules. In order to support this, it's necessary for modules to be able to declare what kinds of tokens they are able to understand. This isn't currently implemented, and can probably be done better, and may not be the best design anyway, so probably okay to remove regardless, as long as it's really not being used. Might want to double-check that callers aren't now getting the parent class' implementation of this method instead, though.

Reply:
Nothing is using the parent method in the Authenticator interface.  That was why I removed it from the interface and implementation.  If you think it is something that should be implemented fully then I can open a ticket.

Reply:
I'm a bit surprised it wasn't used at all... but since it's on the interface, maybe it's best left in place for now, so it doesn't impact implementers... at least until we can rethink the whole design of these pluggable pieces anyway.

Reply:
Reverted in 1da4d9ffaf94991600974b4847b0fdaef70a1d44